angularjs-portal-frame/src/main/webapp/index.jsp

```diff
@@ -1,2 +1,3 @@
 <%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
-<c:import url="/WEB-INF/classes/frame.html" />
+<jsp:include page="/WEB-INF/classes/frame.jsp" />
```

Comment:
So we _could_ just not have 2 JSPs at this point, and move the content of `frame.jsp` to `index.jsp`.  Thoughts?